cli/commands.go

```diff
@@ -284,25 +279,40 @@ func CreateCluster(c *cli.Context) error {
 
 	/* (2)
 	 * Server
-	 * Create the server node container
+	 * Create the server node containers
 	 */
-	serverContainerID, err := createServer(clusterSpec)
-	if err != nil {
-		deleteCluster()
-		return err
+	serverContainersIDs := []string{}
+	for i := 0; i < c.Int("servers"); i++ {
+		if i == 0 {
+			clusterSpec.ServerArgs = append(clusterSpec.ServerArgs, "--cluster-init")
```

Comment:
Setting up a cluster with more than one server node fails for me, because the secondary server nodes still have the `--cluster-init` flag (Log: `level=fatal msg="node ID 1 is in use, please restart"`).
That's probably because the `--cluster-init` flag is appended to struct field here and not removed afterwards for the following server nodes.